jsonschema-core.xml

```diff
@@ -1830,6 +1830,48 @@
                 </section>
             </section>
 
+            <section title="Compound Documents">
+                <t>
+                    A Compound Schema Document is defined as JSON document (sometimes called a "dereferenced" or
+                    "bundled" schema) which has multiple embedded JSON Schema Resources bundled into the same
+                    document to ease transportation.
```

Comment:
I feel like "bundled" is fine here but "dereferenced schema" has other meanings (like the result of dereferencing a single schema) so I'm a little wobbly on that term in this context.